server/cmd/dcrdex/sample-dcrdex.conf

```diff
@@ -0,0 +1,179 @@
+; logdir=~/.dcrdex/logs
+
+; ------------------------------------------------------------------------------
+; Logs settings
+; ------------------------------------------------------------------------------
+
+; Logging level {trace, debug, info, warn, error, critical}
+; debuglevel=info
+
+; Use local time zone time stamps in log entries.
+; loglocal=true
+
+; The number of zipped log files created by the log rotator to be retained. 
+; Setting to 0 will keep all. Default is 32.
+; maxlogzips=32
+
+; ------------------------------------------------------------------------------
+; Network settings
+; ------------------------------------------------------------------------------
+
+; Use testnet.
+; testnet=true
+
+; Use simnet.
+; simnet=true
+
+; ------------------------------------------------------------------------------
+; RPC server settings
+; ------------------------------------------------------------------------------
+
+; RPC server TLS certificate file.
+; Relative to --appdata or absolute path.
+; rpccert=rpc.cert
+
+; RPC server TLS private key file.
+; Relative to --appdata or absolute path.
+; rpckey=rpc.key
+
+; IP addresses on which the RPC server should listen for incoming connections.
+; Default is 127.0.0.1:7232. 
+; rpclisten=127.0.0.1:7232
+
+; A list of hostnames to include in the RPC certificate (X509v3 Subject 
+; Alternative Name)
+; altdnsnames=
+
+; ------------------------------------------------------------------------------
+; Registration fee settings
+; ------------------------------------------------------------------------------
+
+; The extended public key for deriving Decred addresses to which DEX 
+; registration fees should be paid.
+; regfeexpub=
+
+; The number of confirmations required to consider a registration fee paid.
+; Default value is 4.
+; regfeeconfirms=4
+
+; The registration fee amount in atoms.
+; Default value is 1 DCR = 1e8 atoms.
+; regfeeamount=100000000
+
+; ------------------------------------------------------------------------------
+; PostgreSQL settings
+; ------------------------------------------------------------------------------
+
+; PostgreSQL DB name.
+; Default value is network specific.
+; pgdbname=dcrdex_{network}
+
+; PostgreSQL DB user.
+; Default value is "dcrdex"
+; pguser=dcrdex
+
+; PostgreSQL DB password.
+; pgpass=
+
+; PostgreSQL server host:port or UNIX socket (e.g. /run/postgresql).
+; Default value is 127.0.0.1:5432
+; pghost=127.0.0.1:5432
+
+; Logs the PostgreSQL db configuration on start up.
+; showpgconfig=false
+
+; ------------------------------------------------------------------------------
+; Admin server settings
+; ------------------------------------------------------------------------------
+
+; Turn on the admin server.
+; adminsrvon=false
+
+; Administration HTTPS server address.
+; Default value is 127.0.0.1:6542.
+; adminsrvaddr=127.0.0.1:6542
+
+; Admin server password. INSECURE. Do not set unless absolutely necessary.
```

Comment:
I think we can eliminate the "INSECURE" warning.  A sysadmin understands the implications of putting a password in a text file - it has to be secured.
Also please note that if it is not set, then dcrdex will prompt "Admin interface password: "